babynames.py

```diff
@@ -45,9 +45,29 @@ def extract_names(filename):
     with the year string followed by the name-rank strings in alphabetical order.
     ['2006', 'Aaliyah 91', Aaron 57', 'Abagail 895', ' ...]
     """
-    names = []
-    # +++your code here+++
-    return names
+    with open(filename) as f:
+        open_file = f.read()
```

Comment:
Instead of naming this variable `open_file`, use something closer to what it contains. For example, `file_contents` or simply `html_text`